plugins/rts54hub/fu-rts54hub-rtd21xx-device.c

```diff
@@ -357,6 +368,7 @@ fu_rts54hub_rtd21xx_device_detach (FuDevice *device, GError **error)
 static gboolean
 fu_rts54hub_rtd21xx_device_attach (FuDevice *device, GError **error)
 {
+#if 0
```

Comment:
How does the rtd21xx get back into runtime mode?

Reply:
this i2c_write is for changing slave addr 0x3A(ISP mode ) back to 0x6A(normal mode)
But now I move this flow to after read version so it will work fine,
and after update FW are going to reset, it will be in normal mode and work fine too

In original case, after update FW and reset, in that time slave addr is 0x6A(Normal mode) but FWUPD will call fu_rts54hub_rtd21xx_device_attach it will use 0x3A to send i2c_write, it always be stalled by rtd21xx

Reply:
@AnyProblem I think the point is this shouldn't be committed as `#if 0` code - that's indicative of a problem that needs to be explained.  so if it's intended behavior then the code should be removed instead and a comment left explaining why it's not needed in it's place.

Reply:
@superm1 OK, I got it, I will remove #if 0 code and re-push again